backend/runner/connector.py

```
@@ -0,0 +1,74 @@
+            else:
+                self.is_connected = False
+                logger.exception(f"Failed to connect to Naruon Gateway with unexpected error: {e}")
+
+    async def _listen_loop(self):
+        if not self.connection:
+            return
+        try:
+            while self.is_connected:
+                message = await self.connection.recv()
+                await self.handle_message(message)
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)